build/images/tf_operator/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM centos:7
+FROM registry.access.redhat.com/ubi8-dev-preview/ubi:latest
```

Comment:
Sorry, could you please explain why we need to update the base image?

Reply:
This image will effectively replace centos:7 as a preferred community image. Scanning of the centos:7 image revealed some vulnerabilities that are resolved by this newer image.